.github/workflows/test.yaml

```diff
@@ -0,0 +1,38 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  non-Windows:
```

Comment:
Can we unify them by using `if: matrix.os == 'windows-latest'` and `if: matrix.os != 'windows-latest'`?

```yaml
- uses: ruby/setup-ruby@v1
  if: matrix.os != 'windows'
  with:
    ruby-version: ${{ matrix.ruby }}
    bundler-cache: true
- uses: ruby/setup-ruby@v1
  if: matrix.os == 'windows-latest'
  with:
    ruby-version: ${{ matrix.ruby }}
- run: ridk exec bundle install
  if: matrix.os == 'windows-latest'
```

BTW, do we really need `ridk exec` on Windows?

Reply:
I see.
Now I tested it on my fork: e411ed6e9d9188d4ecf1d69f4ab980bebd18085a
And I have confirmed we don't need `ridk exec`: https://github.com/daipom/fluent-plugin-flowcounter-simple/actions/runs/4614256027

Sorry, I don't remember it well, but I remember that some plugin tests fail if we don't use `ridk exec`.
It may be necessary depending on the gem that the plugin depends on.

Reply:
Unified for this repository.
I will check some other repositories that I have created a PR about CI recently.

Reply:
> Sorry, I don't remember it well, but I remember that some plugin tests fail if we don't use ridk exec.
It may be necessary depending on the gem that the plugin depends on.

Sorry, I used `ridk` here since several other plugins used it for CI on Windows.
We don't need it if the plugin doesn't depend on a gem such that it needs some builds.

For this plugin, we don't need `ridk`.